cactusbot/handlers/command.py

```diff
@@ -89,6 +89,9 @@ def __init__(self, channel, api):
 
         json = await response.json()
 
+        if not json["data"]["attributes"].get("enabled"):
```

Comment:
This should be moved to after L104 (`json = ...`), so that in case an alias was recalled, we can check for enabled/disabled, too.